### ISSUE TITLE
module remains in state "loading" if maxUnreachableDepartures > 0

### ISSUE THREAD

**Issue by khassel** [raywo/MMM-PublicTransportHafas#38]
Hi Ray,

example config:

```
{
  module: "MMM-PublicTransportHafas",
  position: "bottom_center",

  config: {
    stationID: "008012202",  // Replace with your stationID!
    stationName: "bla bla",  // Replace with your station name!

    direction: "",                    // Show only departures heading to this station. (A station ID.)
    ignoredLines: [],                 // Which lines should be ignored? (comma-separated list of line names)
    excludedTransportationTypes: [],  // Which transportation types should not be shown on the mirror? (comma-separated list of types) possible values: StN for tram, BuN for bus, s for suburban
    timeToStation: 1,                 // How long do you need to walk to the next Station?
    timeInFuture: 15,                 // Show departures for the next *timeInFuture* minutes.

    showColoredLineSymbols: true,     // Want colored line symbols?
    useColorForRealtimeInfo: true,    // Want colored real time information (timeToStation, early)?
    showTableHeadersAsSymbols: true,  // Table Headers as symbols or text?
    maxUnreachableDepartures: 2,      // How many unreachable departures should be shown?
    maxReachableDepartures: 7,        // How many reachable departures should be shown?
    customLineStyles: "leipzig",      // Prefix for the name of the custom css file. ex: Leipzig-lines.css (case sensitive)
    showOnlyLineNumbers: false,        // Display only the line number instead of the complete name, i. e. "11" instead of "STR 11"

    updatesEvery: 60,
    marqueeLongDirections: false,
  }
},

```

**Comment by raywo** [comment]
Sometimes the HAFAS-API is slow. In this case the module remains in loading state.
Please try again and report it this problem persists.

**Comment by raywo** [comment]
I've found the problem! Your setting of `timeInFuture` is to low. Please set it to a higher value like `60` or so. That way the module will have enough departures to ”play with“.
See also issue #39.

**Comment by raywo** [comment]
Solved with PR #41.
